core/src/main/java/com/novoda/espresso/TalkBackActivityTestRule.java

```diff
@@ -1,26 +0,0 @@
-package com.novoda.espresso;
-
-import android.app.Activity;
-import android.support.test.rule.ActivityTestRule;
-
-public class TalkBackActivityTestRule<T extends Activity> extends ActivityTestRule<T> {
```

Comment:
this and TalkBackViewTestRule (which had some duplicated code) are now defunct: clients can compose rules as they need.